app/handlers/sessions_trusted_launch.rb

```diff
@@ -0,0 +1,32 @@
+class SessionsTrustedLaunch
+
+  lev_handler
+
+  protected
+
+  def authorized?
+    true
+  end
+
+  def handle
+    signup_state = options[:signup_state]
+    uuid_link = UserExternalUuid.find_by_uuid(signup_state.trusted_data['uuid'])
+    if uuid_link.present?
+      outputs[:user] = uuid_link.user
```

Comment:
I don't know for sure but it seems like it might be a bad idea to change user's info after they've signed up.  I'm envisioning a case where an institution has bad info for a user in the LMS (misspelled name or something).  

The user might not be able to edit their records there to correct it, but they would fix it in Tutor, only to have their changes overwritten.

Of course I have no idea how likely that is, but seems like it at least possible.